src/main/java/vazkii/patchouli/client/book/page/PageSpotlight.java

```diff
@@ -14,20 +15,22 @@
 import vazkii.patchouli.client.book.page.abstr.PageWithText;
 import vazkii.patchouli.common.util.ItemStackUtil;
 
+import java.util.List;
+
 public class PageSpotlight extends PageWithText {
 
 	IVariable item;
 	String title;
 	@SerializedName("link_recipe") boolean linkRecipe;
 
-	transient ItemStack itemStack;
+	transient List<ItemStack> itemStacks;
```

Comment:
is it possible to use an `Ingredient` here and use the existing serializer, rather than making a separate serializer for itemstack lists? this isn't ideal because the serializer registry doesn't know about generic types

Reply:
so registering a List.class serializer precludes the usage of any other kind of `List`

Reply:
changed to `Ingredient`